portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java

```diff
@@ -122,7 +122,7 @@ public List<AssetTag> checkTags(long userId, Group group, String[] names)
 		List<AssetTag> tags = new ArrayList<>();
 
 		for (String name : names) {
-			AssetTag tag = fetchTag(group.getGroupId(), name);
+			AssetTag tag = fetchTag(group.getGroupId(), StringUtil.toLowerCase(name));
```

Comment:
In line 134 it also uses the variable `name`. Can you refactor it?

Reply:
Line 134 does not need to be refactored, because the `name` will be processed inside the `addTag` method. I have just modified the line code 125 to trim the `name` tag.